src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/EnvironmentSetupHelper.cs

```diff
@@ -395,6 +395,7 @@ public virtual Collection<PSObject> RunPowerShellTest(params string[] scripts)
         private void SetupPowerShellModules(System.Management.Automation.PowerShell powershell)
         {
             powershell.AddScript("$error.clear()");
+            powershell.AddScript("Set-ExecutionPolicy Bypass -Scope Process");
```

Comment:
Remove this. It impacts all tests run by all RPs. It is not necessary if you set up your machine to allowpowershell scripting.